e2e/support/config.js

```diff
@@ -104,6 +107,7 @@ const defaultConfig = {
       ...dbTasks,
       ...verifyDownloadTasks,
       removeDirectory,
+      signJwt,
```

Comment:
Do we have to sign the token while already on the custom domain?

Reply:
My question was, can we load localhost, sign the jwt, and then navigate to the custom domain?

Reply:
If using the previous method, no. The previous method works on Storybook side via `fetchRequestToken`. So we need to load the storybook completely before the JWT signing kicks in.

Even if we load localhost, sign, and load a custom domain, this function would run again.

To prevent that, we probably need some kind of mechanism to save and reuse the token and then disable `fetchRequestToken` when we visit the custom domain. In that case, I think it might even be more complicated to do that than what I did here.

I'm not sure if that answers your question. Please let me know if I should clarify more.

Reply:
I see, the cypress test is just loading storybook with an aliased domain, and it's the story decorator handling the auth

Reply:
That is correct, that's what we've been doing before this PR.